sdk/runtime/runenv.go

```diff
@@ -99,14 +100,24 @@ type RunEnv struct {
 	RunParams
 	*logger
 
-	unstructured chan *os.File
-	structured   chan *zap.Logger
+	MetricsPusher *push.Pusher
+	unstructured  chan *os.File
+	structured    chan *zap.Logger
 }
 
 // NewRunEnv constructs a runtime environment from the given runtime parameters.
 func NewRunEnv(params RunParams) *RunEnv {
 	re := &RunEnv{
-		RunParams:    params,
+		RunParams: params,
```

Comment:
These groupings cause the metrics to have searchable fields, so I have just added a bunch of them here.

The metric with this configuration looks like this:
```
db_backup_records_processed{TestCase="prometheus",TestCaseSeq="",TestCommit="",TestGroupID="single",TestPlan="example",TestRun="626439704b90",TestTag="",instance="",job="testground/plan"} 42
```